tensorflow/lite/python/interpreter.py

```diff
@@ -290,3 +290,6 @@ def invoke(self):
 
   def reset_all_variables(self):
     return self._interpreter.ResetVariableTensors()
+
```

Comment:
This method deserves documentation.

Reply:
Thanks for the review. Added simple description, rebased, and pushed.